tests/unit/mixins/route-test.js

```diff
@@ -299,6 +289,20 @@ test('it uses extra params when loading more data', function (assert) {
   assert.ok(this.model.get('reachedInfinity'), 'Should reach infinity');
 });
 
+test('route accepts bound params and sets on infinity model to be passed on subsequent requests', function (assert) {
+  assert.expect(3);
+
+  const boundParam = { category: 'myCategory' };
+  this.createRoute({ extra: 'param' }, boundParam);
+
+  assert.deepEqual(this.model.get('_deprecatedBoundParams'), boundParam, '_deprecatedBoundParams');
+
+  this.loadMore();
+
+  assert.deepEqual(this.model.get('_deprecatedBoundParams'), boundParam, '_deprecatedBoundParams');
+  assert.ok(this.model.get('reachedInfinity'), 'Should reach infinity');
+});
```

Comment:
Can we also do a test that essentially checks that when boundParams is passed the `model` can detect the mixin?
https://stackoverflow.com/questions/23922695/how-to-know-if-controller-implements-mixin